Tests/test.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+from dojo import Dojo
+
+
+class TestRoomCreation(unittest.TestCase):
+
+    """ Test cases to for successful room creation """
+
+    def setup(self):
+        self.dojo = Dojo()
+
+    def test_can_create_office(self):
+        self.assertEqual(len(self.dojo.offices), 0)
+        self.dojo.create_room('office', 'marylin')
+        self.assertEqual(len(self.dojo.offices), 1, 'An office has been successfully created!')
+
+    def test_can_create_livingspace(self):
+        self.assertEqual(len(self.dojo.livingspaces), 0)
+        self.dojo.create_room('livingspace', 'manson')
+        self.assertEqual(len(self.dojo.livingspaces), 1, 'A living space has been successfully created!')
+
+    def test_can_create_multiple_office(self):
+        self.assertEqual(len(self.dojo.offices), 0)
+        self.dojo.create_room('office', ['Texas', 'Alabama', 'Chicago', 'Memphis'])
+        self.assertEqual(len(self.dojo.offices), 4, 'offices has been successfully created!')
+
+    def test_can_create_multiple_livingspace(self):
+        self.assertEqual(len(self.dojo.livingspaces), 0)
```

Comment:
@jimmykimani It was the first string of names to pop into my head... the outcome in mind was 
`kevin has been allocated living space kenya`